<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="robot_ip_1" default="10.0.126.35" />
    <node pkg="rosaria"
          type="RosAria"
          name="amigobot_1"
          args="_port:=$(arg robot_ip_1):8101"
          output="screen">
        <param name="DriftFactor" value="0"/>
        <param name="TicksMM"     value="123"/>
        <param name="RevCount"    value="11548"/>
        <param name="odom_frame"      value="/amigobot_1/odom"/>
        <param name="base_link_frame" value="/amigobot_1/base_link"/>
    </node>

    <arg name="robot_ip_2" default="10.0.126.36" />
    <node pkg="rosaria"
          type="RosAria"
          name="amigobot_2"
          args="_port:=$(arg robot_ip_2):8101"
          output="screen">
        <param name="DriftFactor" value="0"/>
        <param name="TicksMM"     value="123"/>
        <param name="RevCount"    value="11548"/>
        <param name="odom_frame"      value="/amigobot_2/odom"/>
        <param name="base_link_frame" value="/amigobot_2/base_link"/>        
    </node>

    <!--arg name="robot_ip_3" default="10.0.126.32" />
    <node pkg="rosaria"
          type="RosAria"
          name="amigobot_3"
          args="_port:=$(arg robot_ip_3):8101"
          output="screen">
        <param name="DriftFactor" value="0"/>
        <param name="TicksMM"     value="123"/>
        <param name="RevCount"    value="11548"/>
    </node-->

    <!--node pkg="amigobot_LTL" name="amigobot_LTL" type="main_w_obtained_trajectories.py" output="screen">
    </node-->

    <node pkg="rviz" name="rviz" type="rviz" output="screen" args="-d /home/ubuntu484/catkin_ws/src/amigobot_LTL/launch/rviz/multi_turtlebot3.rviz">
    </node>

    <!--3.2 0-->
    <node pkg="tf" type="static_transform_publisher" name="amigobot_1_map_odom_broadcaster" args="0 0 0 0 0 0 /map /amigobot_1/odom 5" />
    <node pkg="tf" type="static_transform_publisher" name="amigobot_2_map_odom_broadcaster" args="0 0 0 0 0 0 /map /amigobot_2/odom 5" />


</launch>
